.github/workflows/ci.yml

```diff
@@ -111,18 +111,44 @@ jobs:
           python -m pip install -r requirements-ci.txt
       - name: analyze example session
         timeout-minutes: 5
+        if: always()
         run: |
           . testenv/bin/activate
           ./examples/00_getting_started.py local.localhost
-          SID=$(ls -rt | grep rp.session)
-          echo "$SID: $SID"
+          SID=$(find . -name "rp.session*" -type d -exec basename {} "rp.session" \;)
+          echo "SID analyse example session: $SID"
           radical-analytics-inspect "$SID"
           mkdir artifacts
           ls -la
           cp -R *.png *.stats artifacts
```

Comment:
should we use a single directory for all outputs? thus not to have `artifacts`, but keep it within `sessions_analytics`? (-> having here `mkdir -p sessions_analytics` and the same in the next step?)

p.s. and we will skip the step `upload artifacts`

Reply:
I think we may want to have a directory for each type of test so that it makes easier to look only for the data we need to check in presence of a failure. If all the sessions are together, I still need to look for the specific key_name in a (possibly) long list of other sessions.